packages/admin-ui/src/index.ts

```diff
@@ -3,16 +3,17 @@ export * from '@vtex/admin-ui-react'
 export * from '@vtex/admin-ui-hooks'
 export * from '@vtex/admin-ui-util'
 
-export * from './center'
+export * from './next'
```

Comment:
The same pattern of experimental components. The main difference between those is => that the next ones are ready to ship but lack documentation.

Reply:
I think we should use the same pattern (experimental) for both. The difference would be that these components will be shipped to stable faster

Reply:
Don't think so. They don't have an epic, and they don't have missing features. We also don't advocate for their usage.